project/auth.py

```diff
@@ -212,15 +212,18 @@ def ligue_msi_2025_post():
 
     return add_userleague_row(leagueid, leaguename, userid)
 
-@auth.route('/ligue_worlds_2024', methods=['POST'])
+@auth.route('/ligue_worlds', methods=['POST'])
 @login_required
-def ligue_worlds_2024_post():
+def ligue_worlds_post():
     userid = current_user.id
-    leagueid = 4
-    leaguename = "Worlds 2024"
+    leagueid = 7 # It's a guess
+    year = datetime.datetime.now().year
```

Comment:
```suggestion
    year = 2025
```
Je comprends l'idée mais autant le rentrer en dur, là c'est juste un bug à retardement pour le premier janvier 2026.